### PR TITLE
Dashboard carousel

### PR DESCRIPTION
**Pull request by wesleymui**
### Features
- Published and unpublished maps display on dashboard
- Clicking on published `MapCard` directs user to dummy post page
- Clicking on unpublished `MapCard` directs user to editor with unpublished map
- Username on MapCards

### Issues
- Making changes to the map will not save (Implement a save button?)
- Cannot edit a map when unpublished map is opened from dashboard

### To-Do
- Add details to `MapCard` (author, userId)
- Direct user to actual posts

### CODE REVIEW

**Review by PogChan — DISMISSED** [review]
Lgtm

**Review by EdwardNgg — DISMISSED** [review]
Everything looks amazing.

File: server/tests/user.test.ts [hunk 1]
```diff
@@ -5,7 +5,12 @@ import userModel from '../models/user-model';
 import mongoose from 'mongoose';
 import fs from 'fs';
 
-jest.mock('../auth/index');
```
Comment (EdwardNgg) [inline server/tests/user.test.ts:10]: Adding this line back in should fix the errors in the backend tests for now.

**Review by PogChan — DISMISSED** [review]
lgtm didnt remove my tests

**Review by PogChan — APPROVED** [review]
lgtm

**Comment by EdwardNgg** [comment]
All the progress looks amazing.

**Review by EdwardNgg — APPROVED** [review]

**Pull request merged**